lib/ae_mdw/db/contract.ex

```diff
@@ -62,6 +53,22 @@ defmodule AeMdw.Db.Contract do
     end
   end
 
+  @spec aex9_delete_presence(pubkey(), pubkey()) :: :ok
+  def aex9_delete_presence(contract_pk, account_pk) do
```

Comment:
why is delete presence still needed?

Reply:
It's used on a migration to fix the `create_txi= -1`. With this signature turned out to be used only in tests, so it was moved to a new section for test helpers to be used in other test cases (e.g one present in `437-async-task-index-cache`).